sr_hand/scripts/sr_hand/tactile_receiver.py

```diff
@@ -26,16 +26,24 @@
 
 class TactileReceiver():
 
-    def __init__(self):
+    def __init__(self, prefix = ""):
+        """
+        Receives the tactile information from a Shadow Hand
+        @param prefix - prefix for the tactile topic
+        """
         self.tactile_type = self.find_tactile_type()
         self.tactile_state = None
-        
+
+        #appends trailing slash if necessary
+        if prefix[-1] is not "/":
```

Comment:
@toliver  yes, you are right it will throw 
IndexError: string index out of range